AzureCalling/Controllers/CallViewController.swift

```diff
@@ -544,6 +546,16 @@ class CallViewController: UIViewController, UICollectionViewDelegate, UICollecti
         }
     }
 
+    @objc func appOutOfFocus(_ notification: Notification) {
+        print("appOutOfFocus")
+        callingContext.pauseVideo()
+    }
+
```

Comment:
It's better to rename these two functions similar to `appResignActive` and `appAssignActive`. 
In iOS, resign/assign Active means app is no longer in foreground and the opposite